city_scrapers/spiders/stl_forest_park_advisory.py

```diff
@@ -0,0 +1,185 @@
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class StlForestParkAdvisory(CityScrapersSpider):
+    name = "stl_forest_park_advisory"
+    agency = "Parks, Recreation and Forestry"
+    timezone = "America/Chicago"
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+
+    def __init__(self, *args, **kwargs):
+        self.agenda_map = defaultdict(list)
+        self.minute_map = defaultdict(list)
+        super().__init__(*args, **kwargs)
+
+    def start_requests(self):
+        agenda_url = (
+            "https://sites.google.com/a/stlouis-mo.gov/forest-park-master-plan/home"
+        )
+        yield scrapy.Request(
+            url=agenda_url, method="GET", callback=self._get_agenda_urls, priority=100
+        )
+        yield scrapy.Request(
+            url=agenda_url, method="GET", callback=self._get_minute_urls, priority=100
+        )
+
+        meeting_urls = [
+            (
+                "https://www.stlouis-mo.gov/government/departments"
+                "/parks/parks/forest-park-advisory-board.cfm"
+            ),
+            ("https://www.stlouis-mo.gov/events/past-meetings.cfm?department=117"),
+        ]
+        for url in meeting_urls:
+            yield scrapy.Request(url=url, method="GET", callback=self.parse)
+
+    def parse(self, response):
+        for url in self._get_event_urls(response):
+            yield scrapy.Request(url=url, callback=self._parse_event)
+
+    def _get_event_urls(self, response):
+        event_titles = response.xpath(
+            '//*/a[contains(@href, "Event_ID")]/strong/text()'
+        ).getall()
+        if len(event_titles) == 0:
+            event_titles = response.xpath(
+                '//*/a[contains(@href, "Event_ID")]/text()'
+            ).getall()
+        event_urls = response.xpath('//*/a[contains(@href, "Event_ID")]/@href').getall()
+        urls = []
+        for url, title in zip(event_urls, event_titles):
+            if "Forest Park Advisory" in title:
+                urls.append(response.urljoin(url))
+        return urls
+
+    def _parse_event(self, response):
+
+        start = self._parse_start(response)
+        links_key = datetime.strftime(start, "%Y-%m-%d")
+
+        meeting = Meeting(
+            title=self._parse_title(response),
+            description=self._parse_desc(response),
+            classification=BOARD,
+            start=start,
+            end=self._parse_end(response),
+            all_day=False,
+            location=self._parse_location(response),
+            source=response.url,
+        )
+
+        meeting["links"] = []
+
+        if links_key in self.agenda_map.keys():
+            meeting["links"].append(self.agenda_map[links_key])
+
+        if links_key in self.minute_map.keys():
+            meeting["links"].append(self.minute_map[links_key])
+
+        return meeting
```

Comment:
Parse the `status` and `id` for each meeting like [this](https://github.com/stl-public-meetings/city-scrapers-stl/blob/a2a8d545694f37f580d9bd7973f194328e421e58/city_scrapers/spiders/stl_zoning.py#L99-L100).